profiles/coreos/base/package.use

```diff
@@ -58,3 +58,6 @@ sys-fs/squashfs-tools lzo xz
 
 # make the kernel version discoverable via the traditional gentoo symlink
 sys-kernel/coreos-sources symlink
+
+# enable selinux while building coreutils
+sys-apps/coreutils selinux
```

Comment:
Please move this to coreos/amd64/generic/package.use with the other selinux flags.